ide-twig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "namespaces": [
+        { "path": "source" }
+    ]
+}
```

Comment:
This is to autocomplete twig paths and make them clicable for PHPStorm

Reply:
Nice tip, i did not know about this one!

Reply:
Thanks belongs to @martinhujer

It's killer feature for just clicking to all templates with 0-IDE setup.